web3/_utils/method_formatters.py

```diff
@@ -462,6 +462,19 @@ def apply_list_to_array_formatter(formatter: Any) -> Callable[..., Any]:
 
 signed_tx_formatter = type_aware_apply_formatters_to_dict(SIGNED_TX_FORMATTER)
 
+ACCESS_LIST_FORMATTER = type_aware_apply_formatters_to_dict(
```

Comment:
We should use this same formatter for `TRANSACTION_RESULT_FORMATTERS` in this file for the `accessList` field for consistency. 

One question that I had when I added it there is whether or not we want `to_int_if_hex` on the response for the `storageKeys` and `to_hex_if_int` on the request formatter. It may very well be easier to read / write and they are slot locations so they will represent uint256 as far as I know? Thoughts there? I left it alone until now but now we need to make that decision if we add it since it will change the formatting. I'm inclined to keep what you have here since it's an advanced user that is using this anyways but just curious.

Reply:
On second thought, I'm not sure anyone will be plugging these values in by hand. This will be a value that is spit back from this request and they can use it in their transaction so I think we should probably keep `to_hexbytes(64)`.